src/app/services/xml-parsers/bibliography-parser.service.ts

```diff
@@ -0,0 +1,59 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { AppConfig } from 'src/app/app.config';
+import { forkJoin, Observable } from 'rxjs';
+import { shareReplay, map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BibliographyParserService {
+  private editionUrls = AppConfig.evtSettings.files.editionUrls || [];
+  private bibliographicCitations$: Observable<BibliographicCitation[]>;
+
+  constructor(
+    private http: HttpClient,
+  ) {
+    const parser = new DOMParser();
+    const bibliographicCitations: Array<BibliographicCitation> = [];
+    this.bibliographicCitations$ = forkJoin(this.getHttpCallsOBSStream()).pipe(
```

Comment:
Remove `forkJoin` since you have a single call

Reply:
mind you, I have more than one call. More `http` calls because there may be more files per edition.